cardano-1.19.x.yaml

```diff
@@ -60,7 +60,7 @@ packages:
 - Win32-2.6.2.0
 
 - git: https://github.com/input-output-hk/cardano-base
-  commit: df8687488449f71dce3d881800c21e41fe1b7fc1
+  commit: 3e2d92b25c675fd923a07237eb1d0859b29a79d3
```

Comment:
Damn. This completely breaks the linker when building with stack for me. I am trying on a completely clean stack root now because It reminds me of a past issue with stack not correctly figuring out links between same packages of different snapshots.

Reply:
Yeah I think the YAML filename needs to be different so that stack knows that something's changed. But I couldn't think of a good name for a version somewhere between 1.19.0 and 1.20.0.

Reply:
I needed to bump this library so that the `shelley-spec-ledger-test` package would build. This was just to get access to the `TxMetadata` test generators. But that has caused more problems with multiple `Arbitrary` instances of other types. It's just not worth it. I will copy&paste the `Arbitrary` instance.